engine/src/main/java/org/terasology/logic/characters/CharacterComponent.java

```diff
@@ -36,6 +36,10 @@
      */
     public float interactionRange = 5f;
     /**
+     * Specifies whether the character can interact with entities that have an edit screen.
+     */
+    public boolean editMode = false;
```

Comment:
This name seems somewhat ambiguous, and the PR description confuses me a bit - when it is *true* disallows editing? :-)

Did the variable at first get added to `InteractionScreenComponent` then later moved into `CharacterComponent` ?

How about `canEditInteractionScreens` ? `false` means no, change it to `true` to allow the character to edit interaction screens?